test/allocations.jl

```diff
@@ -0,0 +1,40 @@
+import ClimaCore:
+    ClimaCore,
+    slab,
+    Spaces,
+    Domains,
+    Meshes,
+    Geometry,
+    Topologies,
+    Spaces,
+    Fields,
+    Operators
+
+using Test
+
+Nv = 100
+
+vertdomain = Domains.IntervalDomain(0.0, 1000.0; x3boundary = (:bottom, :top))
```

Comment:
this needs to be `Domains.IntervalDomain(Geometry.ZPoint(0.0), Geometry.ZPoint(1000.0), boundary_tags=(:bottom, :top))`, should fix the test failure